sgid_client/util.py

```diff
@@ -45,4 +47,41 @@ def safe_json_parse(json_string: str) -> dict | list | str:
     try:
         return json.loads(json_string)
     except Exception:
-        return json_string
\ No newline at end of file
+        return json_string
+
+
+def isSgidUserinfoObject(data: object) -> bool:
```

Comment:
same comment as above. `is_sgid_userinfo_object`?

Reply:
Fixed in [396ad25](https://github.com/opengovsg/sgid-client-python/pull/42/commits/396ad257e153840628f4ebbbc22b7676f1c37cfd)